### PR TITLE
Update Readme.md for proper setup

### PR DESCRIPTION
**Pull request by baquer**
Update Readme for the proper setup of project.

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -11,11 +11,21 @@ This is a simple (madani based) quran app for iOS (iPhone/iPad).
...
+## Setting up the iOS Project
 
 1. Install [Cocoapods](https://cocoapods.org).
+
 2. Xcode 9.0 (with Swift 4.0).
-3. Clone the project and open Xcode workspace Quran.xcworkspace.
+
+3. Download the _Quran-iOS_ project source. You can do this either by forking and cloning the repository (recommended if you plan on pushing changes) or by downloading it as a ZIP file and extracting it.
+
+4. Navigate to the unzipped folder and run `pod install`.
```
Comment (mohamede1945) [inline README.md]: We don't need to do this step.

**Comment by baquer** [comment]
Okay , but without installing the pod , how one should be able to run the app?

**Comment by baquer** [comment]
Should I close this PR?

**Comment by mohamede1945** [comment]
The pods are installed and committed already as part of the repo. So the `pod install` is not necessary to do, one can just open the project and compile and run it as is.

**Comment by mohamede1945** [comment]
No, you don't need to close the PR, just remove this step, commit, and push. Then, I will merge your change after that.

**Comment by baquer** [comment]
Okay , step no 4 or 3 or both?

**Comment by mohamede1945** [comment]
Just 4

**Comment by baquer** [comment]
Changes, Done.

**Comment by mohamede1945** [comment]
Jazak Allah khyran for the help.

**Pull request merged**